lib/workers/repository/onboarding/branch/create.spec.ts

```diff
@@ -67,7 +67,9 @@ describe('workers/repository/onboarding/branch/create', () => {
         await createOnboardingBranch(config);
         expect(commitFiles).toHaveBeenCalledWith(
           buildExpectedCommitFilesArgument(
-            `${prefix}${CommitMessage.SEPARATOR} ${message}`
+            `${prefix}${CommitMessage.SEPARATOR} ${
+              message.charAt(0).toLowerCase() + message.slice(1)
```

Comment:
Why this is happen now?

Reply:
Originally this refactored was introduced regarding this issue https://github.com/renovatebot/renovate/issues/12062, therefore I guess the right behaviour here will be lowercased first character of commit message subject.